validation/validationUtils.js

```diff
@@ -14,3 +14,17 @@ export function isValidModelId(data) {
   const idRegexp = new RegExp("^[a-fA-F0-9]{24}$");
   return data && typeof data === "string" && idRegexp.test(data);
 }
+
+export function isValidTimezone(tz) {
+  if (!Intl || !Intl.DateTimeFormat().resolvedOptions().timeZone) {
+    throw "Time zones are not available in this environment";
```

Comment:
maybe better don't use `throw` for consistency and find better way for timezone validation

Reply:
We throw an error when can't validate timezone, that is not good just to return false